experimental/PHP/Dockerfile

```diff
@@ -0,0 +1,47 @@
+﻿FROM microsoft/iis
+
+#
+# PHP 7.1 x64 running on IIS
+#
+
+ENV PHP C:\PHP
+
+# Install VC Redist 14
+ADD https://download.microsoft.com/download/6/A/A/6AA4EDFF-645B-48C5-81CC-ED5963AEAD48/vc_redist.x64.exe /vc_redist.x64.exe

```

Comment:
Avoid ADD. See best practices- https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy

Reply:
Also don't leave turd files around. It's best to avoid creating multiple unnecessary layers

Reply:
We could use COPY if we check in all of the dependencies. I see we do that with servicemonitor along with the other dockerfiles. I can go with that approach.

Reply:
Good point, I'll copy all the files into a c:\setup directory and remove it in one step to prevent deleting files one at a time.